pkg/config/shared.go

```diff
@@ -163,9 +163,12 @@ type KeyVault struct {
 func (v KeyVault) KeyVault() (crypto.KeyVault, error) {
 	switch {
 	case v.Static != nil:
-		return cryptoutil.NewMemKeyVault(v.Static), nil
+		kv := cryptoutil.NewMemKeyVault(v.Static)
+		kv.Separator = ":"
```

Comment:
This is needed for the configuration compatibility commitment; default separator is `/` to not collide with IPv6 colons in hosts